packages/oidc-client/src/lib/client.store.ts

```diff
@@ -330,28 +401,24 @@ export async function oidc<ActionType extends ActionTypes = ActionTypes>({
         const wellknown = wellknownSelector(wellknownUrl, state);
 
         if (!wellknown?.end_session_endpoint) {
```

Comment:
I didn't notice this before, but do we need to add a check for `|| !wellknown.ping_end_idp_session_endpoint`? It seems the logout function calls one or the other depending on the server.

Reply:
Hmm, this is actually a good question. This check is more like a quick check that we have a wellknown response, more or less. It's not really intended to be a complete error proof check. I'll have to think on it.